src/interpret.jl

```diff
@@ -653,13 +653,13 @@ function next_line!(stack, frame, dbstack = nothing)
             # With splatting it can happen that we do something like ssa = tuple(#self#), _apply(ssa), which
             # confuses the logic here, just step into the first call that's not a builtin
             while true
-                dbstack[1] = JuliaStackFrame(JuliaFrameCode(frame.code; wrapper = true), frame, pc)
```

Comment:
When Debugger shares the stack with JuliaInterpereter I expect this whole block to go away.